ui/src/kapacitor/containers/KapacitorRulePage.js

```diff
@@ -23,6 +25,8 @@ export const KapacitorRulePage = React.createClass({
       loadDefaultRule: PropTypes.func.isRequired,
       fetchRule: PropTypes.func.isRequired,
       chooseTrigger: PropTypes.func.isRequired,
+      addEvery: PropTypes.func.isRequired,
```

Comment:
I don't think so. This should be tackled in a separate PR, along with the entire Kapacitor/Alerts app. We should consider using codeshift: https://github.com/reactjs/react-codemod